### DISCUSSION TITLE
On 350% display scaling Properties GUI slightly off screen

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#590] [General]
At 350% display scaling on 4K display the top of the Properties GUI is slightly cut off by the top of the screen.
You can still reach the Close button although part is hidden.
It is not a big issue just mentioning it in case it is fixable.

![full screen](https://user-images.githubusercontent.com/48499740/147241947-89839583-8e99-40cc-a0dc-8a0f64d851ad.png)

**Comment by valinet** [comment]
I have worked a bit on the UI, and besides some reorganization and small enhancements, I took a look at this and provided a fix for it as well:

![image](https://user-images.githubusercontent.com/6503598/147306768-d75d1623-2111-485e-8dad-1d2a632eb6a1.png)

Merry Christmas and a happy new year!

P.S. The window buttons (close, maximize, minimize) look weird and that's a `dwm` bug it seems, at this scaling and resolution, it happens with all Win32 apps for which `dwm` draws the frame. Can't do much about it, but hardly an issue as well.

**Reply by chip33** [reply]
Looks and works great, thanks! Merry Christmas and a Happy New Year to you too!
